### DISCUSSION TITLE
Varying soil types by layer?

### DISCUSSION THREAD

**Discussion by NicWayand** [CH-Earth/summa#545] [Q&A] [answered]
I believe SUMMA currently only allows one single soil type per HRU, is this correct?

Anyone working on layered soil types?

One use case would be to represent a road, were we would want asphalt/cement over gravel etc.

**Comment by wknoben** [comment, marked as answer]
As per #513 (currently available on both `master` and `develop`, it should now be possible to specify soil parameters per layer. I'm uncertain iof this extends to being able to specify different `soilTypeIndex` values for each layer in the attributes netCDF file, but it should be possible to specify parameters for each layer in the trialParameters netCDF file. Anything specified in the trialParams overwrites any other values that may have been obtained through the lookup table that relies on `soilTypeIndex` (see: https://summa.readthedocs.io/en/latest/input_output/SUMMA_input/#attribute-and-parameter-files).

This change could use a bit more testing on a wider variety of cases, so please get in touch if you run into any issues.
